script/plugin_webpack_directories.rb

```diff
@@ -17,7 +17,7 @@
   # skip other rails engines that are not plugins
   # TODO: Consider using the plugin registration api?
   if gemfile_in
-    next unless dep =~ plugin_name_regexp
+    next unless dep.to_s =~ plugin_name_regexp
```

Comment:
In https://github.com/theforeman/foreman/pull/5478 I used `dep.name` because it's the same as line 23

Reply:
I think I tested this in the buildroot and this did not always work so I punted to a safer `to_s`